From dba45b1073b01277511a87364607b53046092e49 Mon Sep 17 00:00:00 2001
From: Chen Hanxiao <chenhanxiao@cn.fujitsu.com>
Date: Tue, 11 Dec 2012 17:07:59 +0800
Subject: [PATCH] Introduce device type 'lun'

 Supporting SG_IO commands pass though,
 introduce a new device type 'lun'
 to keep pace with libvirt.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1077232

(crobinso: Add whitespace tweak)

Signed-off-by: Chen Hanxiao <chenhanxiao@cn.fujitsu.com>
(cherry picked from commit a52591410065db25617c4de2e722896b644cd8ef)
---
 virtinst/VirtualDisk.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/virtinst/VirtualDisk.py b/virtinst/VirtualDisk.py
index 54f072d..7aff785 100644
--- a/virtinst/VirtualDisk.py
+++ b/virtinst/VirtualDisk.py
@@ -304,9 +304,10 @@ class VirtualDisk(VirtualDevice):
         CACHE_MODE_WRITEBACK]
 
     DEVICE_DISK = "disk"
+    DEVICE_LUN = "lun"
     DEVICE_CDROM = "cdrom"
     DEVICE_FLOPPY = "floppy"
-    devices = [DEVICE_DISK, DEVICE_CDROM, DEVICE_FLOPPY]
+    devices = [DEVICE_DISK, DEVICE_LUN, DEVICE_CDROM, DEVICE_FLOPPY]
 
     TYPE_FILE = "file"
     TYPE_BLOCK = "block"
-- 
1.9.0

